lib/linguist/languages.yml

```diff
@@ -4343,6 +4343,17 @@ STON:
   tm_scope: source.smalltalk
   ace_mode: text
   language_id: 336
+Svelte:
+  type: markup
+  tm_scope: text.html.svelte
+  group: HTML
+  ace_mode: html
+  codemirror_mode: htmlmixed
+  codemirror_mime_type: text/html
+  color: "#e34c26"
```

Comment:
You don't need to set a color since it has a group; it will automatically take the parent's color.